docs/minutes/cpansec-minutes-2024-06-19.md

```diff
@@ -0,0 +1,71 @@
+# Minutes
```

Comment:
Add the following to the top of the page:
```
---
layout: page
toc: true
meeting_time: June 19th, 2024 16:00 UTC
title: Minutes 2024-06-19
---
```
These are instructions for Google Pages to render the page correctly (also, with the meeting time, we can add a generated ical link like in `cpansec-minutes-2024-05-15.md`! :smile: )